source/appendices/e-audit-requirements-and-tests/02-audit-tests.rst

```diff
@@ -5,1045 +5,391 @@
 
-Audit Test B3-3: Total_Item_Investigations, Unique_Item_Investigations, and Unique_Title_Investigations
-.......................................................................................................
+The auditor MUST request
 
-**IMPORTANT NOTE**: This test is required when investigations can be reported independently of a request. It is not required when all investigations have a matching request. Any exclusion of tests must be confirmed by COUNTER prior to testing and the auditor notified.
+* 50 journal articles with Access_Type Controlled.
+* 50 journal articles with Access_Type Free_To_Read.
 
-The auditor will perform audit tests based on the relevant options detailed below.
+This MUST result in 50 each of Total_Item_Investigations, Total_Item_Requests, Unique_Item_Investigations and Unique_Item_Requests with Access_Type Controlled, and the same again for Access_Type Free_To_Read.
 
-**Option 1**: Content provider offers OA_Gold items in addition to Controlled.
+**Option 4**: Report provider offers Open and Free_To_Read journal items.
 
-The auditor must make a total of 50 item investigations within a subset of books. There must be 25 Investigations of items within a book where the Access_Type is Controlled and 25 investigations of items within a book where the Access_Type is OA_Gold.
+The auditor MUST request
 
-* Each book must have 5 investigations of unique items. This will report 5 Total_Item_Investigations, 5 Unique_Item_Investigations, and 1 Unique_Title_Investigations.
+* 50 journal articles with Access_Type Open.
+* 50 journal articles with Access_Type Free_To_Read.
 
-This must result in:
+This MUST result in 50 each of Total_Item_Investigations, Total_Item_Requests, Unique_Item_Investigations and Unique_Item_Requests with Access_Type Open, and the same again for Access_Type Free_To_Read.
 
-* 25 OA_Gold Total_Item_Investigations and 25 Controlled Total_Item_Investigations being reported in the TR_B3 Standard View.
-* 25 OA_Gold Unique_Item_Investigations and 25 Controlled Unique_Item_Investigations being reported in the TR_B3 Standard View.
-* 5 OA_Gold Unique_Title_Investigations and 5 Controlled Unique_Title_Investigations being reported in the TR_B3 Standard View.
 
-**Option 2**: Content provider does not offer OA_Gold items.
+E.2.5.2 Journal Year of Publication
+'''''''''''''''''''''''''''''''''''
 
-The auditor must make a total of 50 Investigations within a subset of books.
+For journal content, year of publication (YOP) is useful in evaluating usage of archive content.
 
-* Each book must have 5 investigations of unique items. This will report 5 Total_Item_Investigations, 5 Unique_Item_Investigations, and 1 Unique_Title_Investigations.
+The auditor MUST confirm the Year of Publication (YOP) of articles covered in other Title Report audit tests with appropriate and proportionate spot checks covering a minimum of 10% of all Title Report Audit Tests: Journals.
```

Comment:
The reference seems wrong to me.,Tte relevant sections should be listed instead.